### PR TITLE
Iss 800 7

### PR DESCRIPTION
**Pull request by burkmarr**

### CODE REVIEW

File: js/indicia.datacomponents/idc.core.js [hunk 1]
```diff
@@ -752,6 +752,24 @@
       return Math.abs(lon).toFixed(3) + (lon >= 0 ? 'E' : 'W');
     },
 
+     /**
```
Comment (johnvanbreda) [inline js/indicia.datacomponents/idc.core.js:755]: These will need to be merged with the existing lat and lon functions using a format option.

File: js/indicia.datacomponents/jquery.idc.esDownload.js [hunk 2]
```diff
@@ -296,6 +296,11 @@
       $.extend(currentRequestData, columnSettings);
       // Reset.
       rowsToDownload = null;
+      // If there is an associated download template select control,
```
Comment (johnvanbreda) [inline js/indicia.datacomponents/jquery.idc.esDownload.js:299]: Minor comment - just suffixing -select to the control ID results in an ID which doesn't describe that it's a template selector. Maybe -template would be a better suffix? Also less likely to clash with other controls if other features are added to the download.

File: js/indicia.datacomponents/idc.core.js (hunk 1 above)
Comment (burkmarr) [reply js/indicia.datacomponents/idc.core.js:755]: Oh yes - I will take a look.

File: js/indicia.datacomponents/jquery.idc.esDownload.js (hunk 2 above)
Comment (burkmarr) [reply js/indicia.datacomponents/jquery.idc.esDownload.js:299]: I will change.

File: js/indicia.datacomponents/idc.core.js (hunk 1 above)
Comment (burkmarr) [reply js/indicia.datacomponents/idc.core.js:755]: @johnvanbreda - I'm trying to work out where to test these JS field handlers - where do they kick in?

Comment (johnvanbreda) [reply js/indicia.datacomponents/idc.core.js]: Create a [download] pointing to a [source], then set columnsTemplate to a blank string and use addColumns to add a #lat# column. E.g. (untested)
```
[source]
@id=data-to-download

[download]
@source=data-to-download
@columnsTemplate=
@addColumns=<!--[
  {"caption": "Lat", "field": "#lat#},
  {"caption": "Lon", "field": "#lon#}
]-->
```

**Pull request merged**